### PR TITLE
testfile for case515688/515690/516702

### PR DESCRIPTION
**Pull request by yanpzhan**
Pls help to merge.

### CODE REVIEW

File: cases/515688/pv1.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+metadata:
+  name: pv-one
+spec:
+  capacity:
+    storage: 2Gi
+  accessModes:
+    - ReadWriteOnce
+  nfs:
+    path: /sharezyp
+    server: 10.14.6.144
```
Comment (wsun1) [inline cases/515688/pv1.yaml]: What's this?

Comment (yanpzhan) [inline cases/515688/pv1.yaml]: PV are used for pvcs to bound on. If no pv, my pvc will be always pending.

Comment (wsun1) [inline cases/515688/pv1.yaml]: I mean your server ip

Comment (yanpzhan) [inline cases/515688/pv1.yaml]: It's our nfs server for testing pasted in pad env. For testing usage. It should be only used for openshift-qe I think.

**Comment by wsun1** [comment]
And could you create your directory following our rules.Every directory has meaningful name,I suggest you create pv dir under storage.

File: cases/515688/pv1.yaml (hunk 1 above)
Comment (wsun1) [inline cases/515688/pv1.yaml]: No,it's public repo.Anyone could see it.

**Comment by yanpzhan** [comment]
Will check which directory is suitable. I noticed our nfs server ips have also been used in files under directory  persistent-volumes/nfs

**Comment by yanpzhan** [comment]
Moved files into directory storage/nfs.

File: storage/nfs/tc515688/pv1.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+metadata:
+  name: pv-one
+spec:
+  capacity:
+    storage: 2Gi
+  accessModes:
+    - ReadWriteOnce
+  nfs:
+    path: /sharezyp
+    server: 10.14.6.144
```
Comment (wsun1) [inline storage/nfs/tc515688/pv1.yaml]: @jhou1 Could you help review?I'm not sure if our nfs server could be shared in this repo.

**Comment by jhou1** [comment]
@yanpzhan Is it for automation? We already have enough nfs templates, this template has no difference except name, path and server. So I do not think it's necessary to contribute it again.

**Comment by yanpzhan** [comment]
@jhou1 , yes, it's for automation. I know it's better to reuse the files in the repos, and I have ever tried with existing files in the repo, such as https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/persistent-volumes/nfs/nfs-recycle-rwo.json, and  https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/persistent-volumes/nfs/claim-rwo.json, but after added pvc, it failed on deploying sometimes, showing mounting error in the event, so I changed to use my own files to make the automation pass. I'm a little dazzled by those files, it's hard to choose the suitable ones.

**Comment by jhou1** [comment]
@yanpzhan It will fail unless you replace the nfs ip with functional ones. And in these files you have hardcoded ip. If your auto runs on other openstacks or on aws, it will fail. So I'd advise you keep using the templates maintained by us and replace the mount path and IPs which would work.

Also, I'd suggest you to think about running automation on different IaaS, your automation is more likely to fail on aws/gce/BJ openstack.

**Pull request closed without merging**